src/Dfc.CourseDirectory.WebV2/Features/Home/Start.cshtml

```diff
@@ -4,31 +4,110 @@
 
-            <p class="govuk-body">To use this service you need to sign in using DfE Sign-in.</p>
+            <p class="govuk-body">The information entered will be visible to learners searching for opportunities on the National Career Service, <a href="https://nationalcareers.service.gov.uk/find-a-course" target="_blank"> Find a course</a>, <a target="_blank" href="https://www.tlevels.gov.uk/">TLevels.gov.uk</a> and employers searching for training on <a target="_blank" href="https://findapprenticeshiptraining.apprenticeships.education.gov.uk/">Find apprenticeship training for your apprentice</a>.</p>
+
+            <!--  <p class="govuk-body">Use this service to:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+            <li>add new courses to the course directory</li>
+            <li>edit and update courses in the course directory</li>
+            <li>remove courses from the course directory</li>
+        </ul>-->
 
             <govuk-button is-start-button="true" asp-controller="Auth" asp-action="Login" class="govuk-!-margin-top-2 govuk-!-margin-bottom-8">Start now</govuk-button>
+
+
+            <details class="govuk-details" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        How to access Publish to the course directory
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    You will need a <a href="https://pp-coursedirectory.nationalcareers.service.gov.uk/auth/cb" target="_blank">DfE Sign-in</a> account to access Publish to the course directory, you can create an account and request access to your organisation.<p>
+
+                    <p class="govuk-body">Once your request has been approved, contact the <a href="mailto:support@coursedirectory.org.uk "> Course Directory Support Team</a> who will provide access to the service.</p>
+                </div>
+            </details>
+
+            <details class="govuk-details" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        Provider primary contact details
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    Your organisation contact information is updated from The UK Register of Learning Providers (UKRLP) and visible to learners and employers searching for courses, apprenticeships and T Levels.
+
+                    <p class="govuk-body">Your primary contact details should be up to date at all times, missing details may result in information not being diplayed.  You can <a target="_blank" href="https://www.ukrlp.co.uk/">update</a> your primary contact details at any time.</p>
```

Comment:
Was concerned by this double space,  but it is rendered as a single space on the environment -- so will leave as-is.